test/membrane_http_adaptive_stream/integration_test/reference_playlists/audio_multiple_video_tracks/audio.m3u8

```diff
@@ -0,0 +1,39 @@
+#EXTM3U
```

Comment:
All those files should be under `fixtures` directory as a convention

Reply:
Done.